scripts/percona-release.sh

```diff
@@ -136,6 +137,7 @@ PDPXC80_DESC="Percona Distribution for MySQL 8.0 - PXC"
 PPG_DESC="Percona Distribution for PostgreSQL"
 PDMDB_DESC="Percona Distribution for MongoDB"
 PDPS_DESC="Percona Distribution for MySQL - PS"
+PS_DESC="Percona Server for MySQL - PS"
```

Comment:
we are not using PS_DESC as we have PS80_DESC

Reply:
I don't know how, but it seems like I pulled those changes from the main branch (the source [commit](https://github.com/percona/percona-repositories/pull/118/commits/473d6698a3375153fca9505b6ab2a064268ad564)). You may ask @maqeel75 about that one.

Reply:
It's rebased now and free from foreign commits.